go.mod

```diff
@@ -11,4 +11,6 @@ require (
 	gopkg.in/yaml.v2 v2.2.2 // indirect
 )
 
-replace github.com/urfave/cli => github.com/bradrydzewski/cli v0.0.0-20190108225652-0d51abd87c77
+replace github.com/urfave/cli => github.com/bradrydzewski/cli v1.19.1
```

Comment:
I don't think that this should be changed.

Reply:
Do you know the details of the original commit referenced? When I build, I get:

> go: github.com/bradrydzewski/cli@v0.0.0-20190108225652-0d51abd87c77: invalid pseudo-version: does not match version-control timestamp (2017-04-24T18:43:48Z)


This is why I changed it.

Reply:
using my fork should no longer be required since the `Flag.FilePath` feature was merged upstream. https://github.com/urfave/cli/blob/master/docs/v2/manual.md#values-from-files